src/khoj/main.py

```diff
@@ -14,13 +14,16 @@
 
 # External Packages
 import uvicorn
+import django
 from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
+from rich.logging import RichHandler
 import schedule
-import django
 
 from fastapi.staticfiles import StaticFiles
 from rich.logging import RichHandler
+import schedule
+
```

Comment:
Why are these showing up as new imports? None of the new code in this file seems to need these packages